Src/Support/Google.Apis.Auth/ExistingDependencies/GoogleApiException.cs

```diff
@@ -14,6 +14,9 @@
 limitations under the License.
 */
 
+// Note: the only reference to this is in ServiceAccountCredential. If we throw a different exception,
+// e.g. InvalidOperationException, we could remove GoogleApiException, SingleError and RequestError.
```

Comment:
With some of the initiatives around making errors more actionable I'd still expect things to move in the direction of using specific exception types with structured error messages, even for auth endpoints.

Reply:
Hmm. We should work out what to do on that front then; but it's worth at least being aware that this is the only current usage.